src/python/pywraps2_test.py

```diff
@@ -63,6 +63,19 @@ def testS2CellIdGetEdgeNeighborsIsWrappedCorrectly(self):
     neighbors = cell.GetEdgeNeighbors()
     self.assertEqual(neighbors, expected_neighbors)
 
+  def testS2CellIdGetAllNeighborsIsWrappedCorrectly(self):
+    cell = s2.S2CellId(0x6aa7590000000000)
+    expected_neighbors = (s2.S2CellId(0x2ab3530000000000),
```

Comment:
Use a list, not a tuple, for consistency with above and with what it actually returns.

Reply:
It actually returns a tuple:

```python
>>> import pywraps2 as s2
>>> p = s2.S2CellId.FromToken('6aa759')
>>> type(p.GetAllNeighbors(p.level()))
<class 'tuple'>
```

I haven't configured anything special in SWIG to do this – [this is the default behaviour](https://stackoverflow.com/questions/52960876/tuple-is-returned-by-python-wrapper-generated-by-swig-for-a-c-vector).